packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -22,17 +22,13 @@ Requires: (katello-selinux if selinux-policy-targeted)
 
 # start specfile generated dependencies
 Requires: foreman >= %{foreman_min_version}
-Requires: ruby >= 2.5
-Requires: ruby < 3
 BuildRequires: foreman-assets >= %{foreman_min_version}
 BuildRequires: foreman-plugin >= %{foreman_min_version}
 Requires: ruby >= 2.5
 Requires: ruby < 3
 BuildRequires: ruby >= 2.5
 BuildRequires: ruby < 3
-BuildRequires: rubygems-devel
-BuildArch: noarch
-Provides: foreman-plugin-%{plugin_name} = %{version}
```

Comment:
removing those two seems wrong?

Reply:
If you look carefully it's moved around, to lines 59/60. I debated moving it in the template to make the diff smaller.

Reply:
aha! damn it diff view!